reporter-web-app/build.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id "com.moowork.node" version "1.2.0"
```

Comment:
Nit: Use single quotes where no variable interpolation is required.

Reply:
Done.